Fatal error: numpy/arrayobject.h

The following error occurs during the build:
```
 | /mnt/build/tmp/work/riscv64-oe-linux/python3-tensorflow-lite/2.18.0/git/tensorflow/lite/python/interpreter_wrapper/numpy.h:52:10: fatal error: numpy/arrayobject.h: No such file or directory
|    52 | #include "numpy/arrayobject.h"
|       |          ^~~~~~~~~~~~~~~~~~~~~
```

--- a/recipes-framework/tensorflow-lite/python3-tensorflow-lite_2.18.0.bb
+++ b/recipes-framework/tensorflow-lite/python3-tensorflow-lite_2.18.0.bb
@@ -42,7 +42,7 @@ TENSORFLOW_LITE_BUILD_DIR = "${S}/tensorflow/lite/tools/pip_package/gen/tflite_p
 TENSORFLOW_LITE_DIR = "${S}/tensorflow/lite"
 
 PYBIND11_INCLUDE = "${PYTHON_INCLUDE_DIR}/pybind11"
-NUMPY_INCLUDE = "${PKG_CONFIG_SYSROOT_DIR}/${PYTHON_SITEPACKAGES_DIR}/numpy/core/include"
+NUMPY_INCLUDE = "${PKG_CONFIG_SYSROOT_DIR}/${PYTHON_SITEPACKAGES_DIR}/numpy/_core/include"
 
 OECMAKE_SOURCEPATH = "${TENSORFLOW_LITE_DIR}"
 OECMAKE_TARGET_COMPILE= "_pywrap_tensorflow_interpreter_wrapper"

